### DISCUSSION TITLE
get children/descendants pages from specific page ??

### DISCUSSION THREAD

**Discussion by OffLine911** [getformwork/formwork#936] [Q&A]
how we can get children/descendants pages from specific page and display it ??

**Comment by giuscris** [comment]
You could use `$site->findPage()` to get the page wanted and then use the `children()` method, like this:

```php
$site->findPage('route/to/specific/page')->children();
```

Then you can display children pages with a `foreach` loop:

```php
<?php foreach ($site->findPage('route/to/specific/page')->children() as $childPage): ?>
<article>
    <h1><a href="<?= $childPage->uri() ?>"><?= $childPage->title() ?></a></h1>
    <?= $childPage->content() ?>
</article>
<?php endforeach; ?>
```

I hope you find this information useful.

**Comment by OffLine911** [comment]
thanks it worked
